moldesign/interfaces/biopython_interface.py

```diff
@@ -24,6 +24,12 @@
 from moldesign.helpers.pdb import BioAssembly
 
 
+def exports(o):
```

Comment:
Also, the `@toplevel` decorator does basically the same thing, but for exposing functions at the top-level namespace - see, for example, `moldesign.dihedral`, which is actually defined in `moldesign.geom.coords.dihedral`.